src/index.html

```diff
@@ -1,12 +1,536 @@
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="default__body">
+    <header class="default__header header">
+      <a 
+        href="#"
+        class="header__link">
+        <img 
+          src="./images/logo.png" 
+          alt="logo image"
+          class="header__image"
+        >
+      </a>
+      
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a> 
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="default__main section">
+      <div 
+        class="card section__indent"
+        data-qa="card"
+      >
+        <div class="card__image">
+          <img 
+            src="./images/imac.jpeg" 
+            alt="iMac_icon"
+            class="card__photo"
+          >
+        </div>
+
+        <div class="card__descr">
+          <div class="card__modelName">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__modelCode">
+            Product code: 195434
+          </div>
+        </div>
+
+        <div class="card__rating">
+          <div class="card__stars card__stars--stars_4">
+            <div class="card__stars_star"></div>
```

Comment:
```suggestion
        <div class="card__rating">
          <div class="card__stars stars stars--4">
            <div class="stars__star"></div>
```
`stars` is a separate BEM block from the previous task